CMakeLists.txt

```diff
@@ -263,5 +263,27 @@ message(
 # unmodified links to the latest version.
 string(TOLOWER ${CMAKE_SYSTEM_NAME} SPICY_SYSTEM_NAME)
 set(CPACK_PACKAGE_FILE_NAME "${CMAKE_PROJECT_NAME}-${SPICY_SYSTEM_NAME}")
+set(CPACK_BINARY_TZ OFF)
+set(CPACK_BINARY_TGZ ON)
```

Comment:
Think about adding a step checking whether the archives can be installed.

Especially the DEB packager seems to have issues with the soft links for 3rdparty stuff; let's see whether the CMake-related patch in this branch helps already.